src/lint/linter/WebCliLinter.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/**
+ * Uses UberStandard to detect errors and potential problems in JavaScript code.
+ */
+final class WebCliLinter extends ArcanistExternalLinter {
+
+  private $eslintrc;
+
+  public function getInfoName() {
+    return 'WebCli';
+  }
+
+  public function getInfoURI() {
+    return 'https://github.com/uber/standard';
```

Comment:
web-cli delegates to uber-standard and web-cli's url is internal, https://code.uberinternal.com/diffusion/WEWEBC/ . Do you think that'll be more helpful instead?